a3p-integration/proposals/z:acceptance/test-lib/sync-tools.js

```diff
@@ -0,0 +1,285 @@
+/**
+ * @file The purpose of this file is to bring together a set of tool that
+ * developers can use to synchronize operations they carry out in their tests.
+ *
+ * These operations include;
+ * - Making sure a core-eval resulted succesfully deploying a contract
+ * - Making sure a core-eval succesfully sent zoe invitations to committee members for governance
+ * - Making sure an account is successfully funded with vbank assets like IST, BLD etc.
+ *  - operation: query dest account's balance
+ *  - condition: dest account has a balance >= sent token
+ * - Making sure an offer resulted successfully
+ *
+ */
+
+/**
+ * @typedef {object} RetyrOptions
```

Comment:
typo
```suggestion
 * @typedef {object} RetryOptions
```

Reply:
resolved in https://github.com/Agoric/agoric-sdk/pull/10171/commits/009d0c49dc2dbc457982e8effc048cbb1e496d36